profile/README.md

```diff
@@ -0,0 +1,54 @@
+# We are NewAperio
+
+![hero images](https://user-images.githubusercontent.com/1529366/173092313-2938e23d-8e4b-4d93-87f8-c827d38c4002.png)
+
+### What we do:
```

Comment:
I think we should use sequential heading levels if possible, but I know this is tricky since heading level is one of the only ways to control font size. GitHub starts with a seconds-level heading in [their README](https://raw.githubusercontent.com/github/.github/master/profile/README.md), which is… better?